al_aws_collector.js

```diff
@@ -250,6 +250,10 @@ class AlAwsCollector {
     selfUpdate(callback) {
         m_alAws.selfUpdate(callback);
     }
+    
+    selfConfigUpdate(callback) {
```

Comment:
I think we need new method, `update()` which will include both code and config updates.
I think we don't need to have them separately in this class.